core/jobs/batch_jobs/suggestion_stats_computation_jobs.py

```diff
@@ -97,98 +150,301 @@ def run(self) -> beam.PCollection[job_run_result.JobRunResult]:
+            # detect that the value is passed through the pipe.
+            | 'Remove question contribution keys' >> beam.Values()  # pylint: disable=no-value-for-parameter
+            | 'Transform question contribution result to job run result' >> (
+                job_result_transforms.ResultsToJobRunResults())
+        )
+        user_question_review_stats_error_job_run_results = (
+            question_review_stats_keys_and_results
+            | 'Filter question review err results' >> beam.Filter(
+                lambda key_and_result: key_and_result[1].is_err())
+            # Pylint disable is needed because pylint is not able to correctly
+            # detect that the value is passed through the pipe.
+            | 'Remove question review keys' >> beam.Values()  # pylint: disable=no-value-for-parameter
+            | 'Transform question review result to job run result' >> (
                 job_result_transforms.ResultsToJobRunResults())
         )
 
-        unused_put_result = (
-            user_stats_models
-            | 'Put models into the datastore' >> ndb_io.PutModels()
+        unused_contribution_put_result = (
+            user_contribution_stats_models
+            | 'Put contribution models into the datastore' >> ndb_io.PutModels()
+        )
+        unused_review_put_result = (
+            user_review_stats_models
+            | 'Put review models into the datastore' >> ndb_io.PutModels()
+        )
+        unused_question_contribution_put_result = (
+            user_question_contribution_stats_models
+            | 'Put question contribution models into the datastore' >> (
+                ndb_io.PutModels())
+        )
+        unused_question_review_put_result = (
+            user_question_review_stats_models
+            | 'Put question review models into the datastore' >> (
+                ndb_io.PutModels())
         )
 
         user_stats_models_job_run_results = (
-            user_stats_models
-            | 'Create job run result' >> (
+            user_contribution_stats_models
+            | 'Create contribution job run result' >> (
+                job_result_transforms.CountObjectsToJobRunResult())
+        )
+        user_review_stats_models_job_run_results = (
+            user_review_stats_models
+            | 'Create review job run result' >> (
+                job_result_transforms.CountObjectsToJobRunResult())
+        )
+        user_question_contribution_stats_models_job_run_results = (
+            user_question_contribution_stats_models
+            | 'Create question contribution job run result' >> (
+                job_result_transforms.CountObjectsToJobRunResult())
+        )
+        user_question_review_stats_models_job_run_results = (
+            user_question_review_stats_models
+            | 'Create question review job run result' >> (
                 job_result_transforms.CountObjectsToJobRunResult())
```

Comment:
Can you supply the model name to each of the results, e.g. `TranslationContributionStats SUCCESS count`. Right now, the output will just have 4 `SUCCESS: X`, which is not very descriptive.